README.md

```diff
@@ -33,7 +33,9 @@ It supports:
 </script>
 
 <Template let:args>
-  <Button {...args} on:click={handleClick}>
+  <Button {...args} 
+    on:click <-- forwarded for actions
```

Comment:
is this a custom syntax? or can you do this in Svelte?

also, `<--` is pretty close to `<!--` which I read it as first. it might be nice to actually make it a comment like that and expand the text a bit

Reply:
It was just a way to put a comment near the attribute, but yeah, I'll rewrite that ^^